storage/src/block_meta_storage.rs

```diff
@@ -427,26 +420,12 @@ impl Decoder for Meta {
                         let block_hash = bytes
                             [next_successor_index..(next_successor_index + LEN_BLOCK_HASH)]
                             .to_vec();
-                        assert_eq!(
-                            LEN_BLOCK_HASH,
-                            block_hash.len(),
-                            "Successor expected length is {} but found {}",
-                            LEN_BLOCK_HASH,
-                            block_hash.len()
-                        );
                         successors.push(BlockHash::try_from(block_hash)?);
                     }
                 }
                 successors
             };
 
-            assert_eq!(
```

Comment:
This should newer be triggered as the resulting length is  `LEN_CHAIN_ID`, and also correct length is checked below in `chain_id.try_into()?`.